packages/studio/tests/components/ViewportButton.test.jsx

```diff
@@ -0,0 +1,7 @@
+import { render, screen } from "@testing-library/react";
+import ViewportButton from "../../src/components/Viewport/ViewportButton";
+
+it("renders viewport button", () => {
+  render(<ViewportButton />);
```

Comment:
it's actually possible to test that clicking things in the button affects the store! You can do something like 
```ts
expect(useStudioStore.getState().blah.state.whatever).toEqual(something);
```